frontend/src/pages/article/article-page.tsx

```diff
@@ -51,7 +51,7 @@ const ArticlePage: React.FC = () => {
   const { text, title, author } = article ?? {};
 
   return (
-    <Loader isLoading={isLoading}>
+    <Loader isLoading={isLoading} hasOverlay={true} loaderType="circular">
```

Comment:
And if you've created a enum - then why don't you use it?
```suggestion
    <Loader isLoading={isLoading} hasOverlay loaderType="circular">
```